### ISSUE TITLE
Uncaught Error: Call to undefined function GuzzleHttp\Psr7\stream_for()

### ISSUE THREAD

**Issue by MaksBrainiac** [zf-fr/zfr-shopify#124]
PHP 8.1, minimal project

`
"require": {
        "zfr/zfr-shopify": "^6.7"
    }
`

Minimal code:

```
$shopifyClient = new \ZfrShopify\ShopifyClient([
    'private_app' => true,
    'api_key'     => 'e7a11***************,
    'password'    => 'shpss_***************',
    'shop'        => '***************.myshopify.com',
    'version'     => '2022-04'
]);

$details = $shopifyClient->createProduct([
    'title' => 'Test',
]);
```

Exception:

```
Fatal error: Uncaught Error: Call to undefined function GuzzleHttp\Psr7\stream_for() in ...\vendor\zfr\zfr-shopify\src\ShopifyClient.php:517
Stack trace:
#0 ...\vendor\guzzlehttp\command\src\ServiceClient.php(195): ZfrShopify\ShopifyClient->wrapRequestData()
#1 ...\vendor\guzzlehttp\command\src\ServiceClient.php(172): GuzzleHttp\Command\ServiceClient->transformCommandToRequest()
#2 [internal function]: GuzzleHttp\Command\ServiceClient->GuzzleHttp\Command\{closure}()
#3 ...\vendor\guzzlehttp\promises\src\Coroutine.php(71): Generator->current()
#4 ...\vendor\guzzlehttp\promises\src\Coroutine.php(86): GuzzleHttp\Promise\Coroutine->__construct()
#5 ...\vendor\guzzlehttp\promises\src\functions.php(362): GuzzleHttp\Promise\Coroutine::of()
#6 ...\vendor\guzzlehttp\command\src\ServiceClient.php(181): GuzzleHttp\Promise\coroutine()
#7 ...\vendor\guzzlehttp\guzzle-services\src\Handler\ValidatedDescriptionHandler.php(79): GuzzleHttp\Command\ServiceClient->GuzzleHttp\Command\{closure}()
#8 ...\vendor\guzzlehttp\command\src\ServiceClient.php(85): GuzzleHttp\Command\Guzzle\Handler\ValidatedDescriptionHandler->GuzzleHttp\Command\Guzzle\Handler\{closure}()
#9 ...\vendor\guzzlehttp\command\src\ServiceClient.php(77): GuzzleHttp\Command\ServiceClient->executeAsync()
#10 ...\vendor\zfr\zfr-shopify\src\ShopifyClient.php(448): GuzzleHttp\Command\ServiceClient->execute()
#11 ...\vendor\zfr\zfr-shopify\src\ShopifyClient.php(496): ZfrShopify\ShopifyClient->execute()
#12 ...\src\index.php(18): ZfrShopify\ShopifyClient->__call()
#13 {main}
  thrown in ...\vendor\zfr\zfr-shopify\src\ShopifyClient.php on line 517
```

**Comment by MaksBrainiac** [comment]
Can be resolved by this pull request:
https://github.com/zf-fr/zfr-shopify/pull/123

**Comment by MaksBrainiac** [comment]
Fixed in v6.8.0
